connector_cli/oauth2login.py

```diff
@@ -59,6 +61,11 @@ def login_callback():
             "oauth_client_id": client_id,
             "oauth_client_secret": client_secret,
         }
+        if manifest.get("requires_service_api_access"):
+            secrets["service_api_access"] = service_jwt
```

Comment:
I think the secret in this case should be called `service_jwt`, like here https://github.com/datanav/connector-deployer/blob/0bb39b2aae70dec3e821ee657bfa7325b083e33c/service/service.py#L818

Reply:
done